google-cloud-storage/test/helper.rb

```diff
@@ -48,11 +48,9 @@ class MockStorage < Minitest::Spec
     addl.include? :mock_storage
   end
 
-  def random_bucket_hash(name=random_bucket_name,
-    url_root="https://www.googleapis.com/storage/v1", location="US",
-    storage_class="STANDARD", versioning=nil, logging_bucket=nil,
-    logging_prefix=nil, website_main=nil, website_404=nil, cors=[], requester_pays=nil,
-    lifecycle=nil, location_type="multi-region")
+  def random_bucket_hash name: random_bucket_name, url_root: "https://www.googleapis.com/storage/v1", location: "US", storage_class: "STANDARD", versioning: nil,
+                         logging_bucket: nil, logging_prefix: nil, website_main: nil, website_404: nil, cors: [], requester_pays: nil, lifecycle: nil,
+                         location_type: "multi-region"
```

Comment:
I suggest one argument per line. Would be easier to read.

Reply:
Thanks, Daniel. I made that change.